__tests__/data/sample_request_1.json

```diff
@@ -1,3 +1,9 @@
-{
-    "body": "{}"
+{ 
+    "method": "GET",
+    "body": { },
+    "query": {
+        "race": "ELF"
+    },
+    "path": { },
+    "requestPath": "/v1/generate/character"
 }
```

Comment:
This is more like what an actual API Gateway request would be like (without the headers)